tests/testerina-integration-test/src/test/resources/project-based-tests/function-mocking-tests/tests/main_test.bal

```diff
@@ -239,6 +239,14 @@ public function callOriginal_Test1() {
     test:assertEquals(floatAdd(10.6, 4.5), 15.1);
 }
 
+@test:Config {}
+public function callOriginal_Test2() {
```

Comment:
@gabilang I don't see a test case with `thenReturn` followed by `callOriginal` for the same function within a test case. Shall we add that too?

Reply:
For the same function it always going to take the last mock return type right? And it didn't give any issue with previous logic too. The issue observed with different functions when they are used to assert something like follows:
```
test:when(mock_intAdd).thenReturn(100);
test:when(mock_stringAdd).callOriginal();
test:assertEquals(intAdd(10, 6), 100);
test:assertEquals(stringAdd("hello"), "default");
```

Reply:
Yeah you are right. But we had a gap of such a test case here and it's good to have. Thanks for adding :)